src/components/MainLayout/MainPage/ScorePanel/ScorePanelStyled.tsx

```diff
@@ -4,7 +4,6 @@ export const ScorePanelStyled = styled.div`
   display: grid;
   grid-area: left;
   height: 100%;
-  background-image: url(${cardExample});
```

Comment:
This code is unnecessary, because cardExaple doesn't imported from another component. Also background image is unnecessary because the layout of the PlayerCards has already been developed